download_models/getModels.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+REFRESH_INPUT=
+OPEN_OVMS=0
+
+show_help() {
+    echo "
+        usage: $0
+            --workload dlstreamer|opencv-ovms
+            --refresh 
+        Note:
+            --refresh is optional, it removes previously downloaded model files
+    "
+}
+
+get_options() {
+    while :; do
+        case $1 in
+            -h | -\? | --help)
+                show_help
+                exit
+            ;;
+            --refresh)
+                echo "running model downloader in refresh mode"
+                REFRESH_INPUT="--refresh"
+                ;;
+            --workload)
+                echo "workload: ${2}"
+                if [ "$2" == "opencv-ovms" ]; then
+                    OPEN_OVMS=1
+                else 
+                    if [ "$2" == "dlstreamer" ]; then
+                        OPEN_OVMS=0
```

Comment:
this if assignment is redundant since the initial value of `OPEN_OVMS` is 0 ; just use if ! $2 = "dlstreamer"